docs/src/main/asciidoc/parameter-store.adoc

```diff
@@ -105,9 +105,21 @@ enable it you need an additional dependency:
 </dependency>
 ----
 
-However, starting at `spring-cloud-aws` `2.3`, allows import default aws' parameterstore keys
-(`spring.config.import=aws-parameterstore:`) or individual keys
-(`spring.config.import=aws-parameterstore:config-key;other-config-key`)
+However, starting at `spring-cloud-aws` `2.3`, allows import default aws parameterstore keys
+[cols="2*", options="header"]
+|===
+|property
+|explanation
```

Comment:
```suggestion
|Description
```

Reply:
Since other column description names in other tables are lower cased should  we upper case them as well?